Decoder/DecoderProvider.php

```diff
@@ -11,21 +11,19 @@
 
 namespace FOS\RestBundle\Decoder;
 
-use Symfony\Component\DependencyInjection\ContainerAware;
-
 /**
  * Provides encoders through the Symfony2 DIC.
  *
  * @author Igor Wiedler <igor@wiedler.ch>
  */
-class ContainerDecoderProvider extends ContainerAware implements DecoderProviderInterface
+class DecoderProvider implements DecoderProviderInterface
```

Comment:
Well, maybe we should inject the container in the constructor as it is required anyway. What do you think?